scripts/deploy-bitte

```diff
@@ -16,7 +16,8 @@ then
     error "GITHUB_SHA environment variable not set!"
 fi
 
-readonly ref="${GITHUB_REF_SLUG:-}"
+ref="${GITHUB_REF_SLUG:-}"
+ref="${ref}sprint-${[0-9]} | sed -e 's/[0-9]//g'"
```

Comment:
This isn't right. You can test this out locally by doing:

```
$ ref=sprint-23
$ ref=<expression you want to test>
$ echo $ref
```

I recommend just using the bash replacement style like I did in plutus-apps. See https://tldp.org/LDP/abs/html/string-manipulation.html#SUBSTRREPL00

Reply:
ok I think I got it now - the new regex works locally. Thanks for your help on this @shlevy